### PR TITLE
(WIP) .950 JDJ and the Compensator (put up because I can't test it myself, and need others to.)

### PR DESCRIPTION
**Pull request by MicroMinty**
Big guns are nice, aren't they?

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>
A work-in-progress big-ass gun, which won't be implemented most likely, but if it is, there will be a bluecross variant before I push it. I only have this up because Moon offered to test it for me, due to an inability to run a testserver on my end.

	
<hr>
</details>

## Changelog
:cl:
add: Added Artificer's guild "Compensator" elephant rifle.
add: Added .950 JDJ cartridge, only basic lethals for now.
add: Added CAL_ELEPH def, for the massive rounds to be able to be distinguished.
imageadd: Added sprites for on ground elephant rifle, both open and not.
imageadd: Added sprites for .950 JDJ lethal
/:cl:

### CODE REVIEW

**Comment by MicroMinty** [comment]
Do NOT merge this

**Comment by MicroMinty** [comment]
Wow that's a lot of errors

**Comment by MicroMinty** [comment]
i think im more retarded than i realised

**Comment by MicroMinty** [comment]
hheyyyyyyy i think i fixed them now?

**Comment by cdb-is-not-good** [comment]
Im not TOO sure we want to add a brand new caliber for just one gun

**Comment by Trilbyspaceclone** [comment]
New caliber is a big *sigh* sprite looks good imo

**Comment by SouthyOsmanli** [comment]
This is effectively an AMR but better with a new caliber. I'm - not sure I'd suggest this since the Bat'ko itself is already a 'lesser' AMR, and both the AMR and Bat'ko can shoot limbs off (AMR is 1-2 shots, Bat'ko in roughly 2-3).

I'd more suggest to make it either use an existing caliber or be a matter gun, where you load certain paths into it as types of ammo. Ex how the Bat'ko works effectively. Even then - I'd still say that projectile is WAY too strong to ever want to use anything else.

**Comment by lolman360** [comment]
this can honestly just be a boltgun with 1 ammo capacity (like the kragjorgenson), and blacklist the overshooter if it's really a concern

otherwise yes this is hilariously overpowered even if you had to handmake the ammo, inbetween baroque and AMR is probably a decent niche for it

**Comment by MicroMinty** [comment]
> 

i would do that but im retarded

**Comment by MicroMinty** [comment]
and much less silly

**Comment by lolman360** [comment]
> > 
> 
> i would do that but im retarded - draft it, that is. This was mostly just to see if I could successfully implement a new calibre and have it 'work'. it'll be .60-06 when i finish

why not 30mm? (baroque) 
it'd be good to have a not-lodge weapon chambered in it

**Comment by Chevalt** [comment]
I absolutely do not mind something that does an incredible amount of damage, for the balance of it, you'd only really need to make so it needs to be set up on the ground or on a table before firing, this would make it on the level of most weapons as it wouldn't be possible to use it in regular PVP easily.

File: code/modules/projectiles/ammunition/bullets.dm [hunk 1]
```diff
@@ -795,3 +795,11 @@
 	name = "flashforged rod"
 	desc = "Wait a second, this is a flashforged bullet!"
 	projectile_type = /obj/item/projectile/bullet/rod_bolt/rcd
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/ammunition/bullets.dm:797]:
```suggestion
	projectile_type = /obj/item/projectile/bullet/rod_bolt/rcd

```
Just line stuff not a real issue

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 2]
```diff
@@ -824,3 +824,13 @@
 	affective_damage_range = 7
 	affective_ap_range = 7
 	create_type = null
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:826]:
```suggestion
	create_type = null

```
Just line spaces

**Comment by Churro55** [comment]
We REALLY don't need yet another AMR-like weapon, much less Guild craftable. That'd be like what, the third, fourth? Those were a pain to get balanced and still aren't balanced.
It's even better than the AMR in raw damage. Hitscan too. (In it's current form)

**Comment by MicroMinty** [comment]
> Late, I know.
> 
> I'd prefer if your cartridge was renamed from a literal IRL sporting cartridge. Something else would be fine, but .950 JDJ is explicit to SSK Industries and, the guy who made it, in naming. That's the initials of the guy who made it, JDJ. The sprite also reminds me vaguely of the SSK Industries firearm that shoots it, just made with wooden furnishings instead -- barring the fact that it's a break action loader, instead of a bolt action (even though it LOOKS like it'd open at the bolt on the back end of the barrel?). I don't mind it too much it's interesting enough I suppose. Should probably uniform the sprite based on whether you want it to be break action or bolt action.
> 
> Also where'd the scope on the in-hands come from?

all of the sprites are just stand-ins for when i finished the code it's self, and the in-hands are just stolen from one of the rifles.  THe cartridge would be .60-06 when finished, the new calibre was just to see if i could get it to work.

Also, 
> We REALLY don't need yet another AMR-like weapon, much less Guild craftable. That'd be like what, the third, fourth? Those were a pain to get balanced and still aren't balanced.
> It's even better than the AMR in raw damage. Hitscan too. (In it's current form)

I was already planning to either change the damage or make it a bluecross and rename it. 

EITHER WAY, though, im closing this pr for now so i can work on less stupid stuff!

**Pull request closed without merging**